lib/studentscreen.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+
+class StudentScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.white,
+
+      appBar: AppBar(
+        title: Text("Student"),
+        backgroundColor: Color.fromRGBO(75,209,160, 1),
+      ),
+      body: Center
+      (
+        
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          
+          children:[
+            Text(
+              "Student Name Here",
+              textScaleFactor: 2,
+              textAlign: TextAlign.justify,
+            ),
+            Text(
+              "Other Stuff",
+              textScaleFactor: 2,
+              textAlign: TextAlign.justify,
+            ),
+            RaisedButton(
+              onPressed: () {
+                Navigator.pop(context);

```

Comment:
This is fine for now but we should move to using named routes ASAP: https://flutter.dev/docs/cookbook/navigation/named-routes